build.sbt

```diff
@@ -67,13 +67,12 @@ lazy val mimaSettings = mimaDefaultSettings ++ Seq(
   mimaPreviousArtifacts := {
     val VersionPattern = """^(\d+).(\d+).(\d+)(-.*)?""".r
     val previousVersions = version.value match {
+      case VersionPattern("2", "0", "5", _) if scalaBinaryVersion.value == "2.13" => Set.empty // added 2.13.0 support in 2.0.5-SNAPSHOT.
       case VersionPattern(epoch, major, minor, _) => (0 until minor.toInt).map(v => s"$epoch.$major.$v")
       case _ => sys.error(s"Cannot find previous versions for ${version.value}")
     }
 
-    val sbv = scalaBinaryVersion.value
-    if (sbv.equals(scala213)) Set.empty
```

Comment:
I removed this since a comparison like `"2.13".equals("2.13.0")` would never be true.